javascript/ql/src/semmle/javascript/security/dataflow/CodeInjectionCustomizations.qll

```diff
@@ -138,4 +138,17 @@ module CodeInjection {
         API::moduleImport("module").getInstance().getMember("_compile").getACall().getArgument(0)
     }
   }
+
+  /**
+   * A construction of a JSDOM object (server side DOM), where scripts are allowed.
+   */
+  class JSDomWithRunScripts extends Sink {
```

Comment:
It would fit better in the XSS query. `js/code-injection` deliberately steps through HTML sanitizers which could give FPs for this sink.